string name # name of the landmark
float64 distance # the distance to the landmark in meters
